practice-two/src/pages/Analytics/Analytics.jsx

```diff
@@ -148,7 +148,7 @@ const Analytics = () => {
       {selectedCustomer && isShowProfileInfo && (
         <ProfileInfo selectedCustomer={selectedCustomer} />
       )}
-      {isShowForm && <FormValidation onShowForm={handleShowForm} />}
+      {isShowForm && <FormValidation onShowForm={handleShowForm}  />}
```

Comment:
two space in here

Reply:
![image](https://github.com/datdragon777/React-Training/assets/74806826/80d6f183-5b6b-4a0a-813d-249a265c5b1d)